src/orderbooks.md

```diff
@@ -0,0 +1,204 @@
+
+While order books on decentralized exchanges offer numerous benefits, it's important to consider their limitations:
+
+- **Scalability**: Blockchains have limitations in processing transaction volumes.
+- **Liquidity**: Liquidity of decentralized order books can be limited.
+- **User experience**: Interfaces may be more complex and less user-friendly compared to centralized solutions.
+- **Fewer trading instruments**: Centralized exchanges typically offer a wider range of order types.
+- **Order book management**: In centralized systems, the exchange operator can manage the order book and ensure its efficient operation. In decentralized order book implementations, management is left to users, which can lead to errors, inefficiencies, and potential manipulation.
+
+## Data model
+
+You can review the list below to gain a deeper understanding of the data models for each component that can be built using this API:
+
+- [Calls](https://sora-xor.github.io/sora2-network/order_book/pallet/enum.Call.html)
+- [Errors](https://sora-xor.github.io/sora2-network/order_book/pallet/enum.Error.html#variants)
+- [Events](https://sora-xor.github.io/sora2-network/order_book/pallet/enum.Event.html#variants)
+- [Storages](https://sora-xor.github.io/sora2-network/order_book/pallet/index.html#types)
+- [More info](https://sora-xor.github.io/sora2-network/order_book/index.html#reexport.UserOrders)
+
+## Integration Example
+
+Here are code snippets written in multiple languages:
+
+::: code-group
+
+```ts [ts]
+import { api } from "@sora-substrate/util";
+import { PriceVariant } from "@sora-substrate/liquidity-proxy";
+
+import { delay, withConnectedAccount } from "./util";
+
+async function main(): Promise<void> {
+  await withConnectedAccount(async () => {
+    const xor =
+      "0x0200000000000000000000000000000000000000000000000000000000000000";
+    const val =
+      "0x0200040000000000000000000000000000000000000000000000000000000000";
+    const account = "cnVkoGs3rEMqLqY27c2nfVXJRGdzNJk2ns78DcqtppaSRe8qm";
+
+    const orderBooks = await api.orderBook.getOrderBooks();
+    console.log("orderBooks", orderBooks);
+
+    await api.orderBook.getUserOrderBooks(account);
+
+    api.orderBook
+      .subscribeOnUserLimitOrdersIds(val, xor, account)
+      .subscribe((ids) => {
+        console.log("ids", ids);
+      });
+
+    const order = await api.orderBook.getLimitOrder(val, xor, 16);
+    console.log("order", order);
+
+    api.orderBook.subscribeOnAggregatedAsks(val, xor).subscribe((asks) => {
+      console.log("asks", asks);
+    });
+
+    api.orderBook.subscribeOnAggregatedBids(val, xor).subscribe((bids) => {
+      console.log("bids", bids);
+    });
+
+    const price = "1100000000000000000";
+    const amount = "100000000000000000000";
+    const side: PriceVariant = PriceVariant.Buy;
+
+    await api.orderBook.placeLimitOrder(val, xor, price, amount, side);
+
+    await delay(100000);
+  });
+}
```

Comment:
@stefashkaa @RustemYuzlibaev could you check that this is still actual?

Reply:
@bgodlin is this addressed?